elastic/kibana
# [Discover] Discover Classic View Filter In/Out placement when `truncate:maxHeight` is set to 0

## Summary

Fixes #128080

When setting `truncate:maxHeight` to `0` in Advanced settings, CSS for truncating height of Classic table was no longer injected in global CSS. 

![Bildschirmfoto 2022-04-13 um 11 46 51](https://user-images.githubusercontent.com/463851/163158817-14938ed4-e470-49bd-9ddc-3a4b1b054f70.png)

This led to an issue with the positioning of the filter buttons. It looked like this

<img width="713" alt="image" src="https://user-images.githubusercontent.com/16760856/159050763-cf4cd497-1eca-43a0-b0dc-d8fc149da803.png">



This PR takes care of injecting CSS also for the `0` case, so the positioning of the filter buttons works correctly:

![Bildschirmfoto 2022-04-13 um 12 16 05](https://user-images.githubusercontent.com/463851/163159314-015b7436-c5aa-4577-8020-52ae99437b25.png)

What's more, since it takes a refresh of the browser page for the change of the `truncate:maxHeight` to take effect, there's now a toast displayed when users persisted the change


**Steps to test:**
1. Enable classic view in Advanced settings by setting `doc_table:legacy` to `On`. 
2. Go to Discover , add a column with long content to the table, check that filter buttons when hovering look correctly
3. Save the search and create a new Dashboard with it.
4. Goto Advanced setting enable Classic table,  set `truncate:maxHeight` to `0`
5. When saving it you should get a toast to reload the page, which you should do. 
6. Go to Discover, check that filter buttons on multiline context are positioned correctly. 
7. Go to Dashboard, check that filter buttons on the Dashboard using the saved search you've created, look correctly


### Checklist

Delete any items that are not applicable to this PR.
